libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -1262,17 +1003,17 @@ void AC_AutoTune::update_gcs(uint8_t message_id)
 }
 
 // axis helper functions
-inline bool AC_AutoTune::roll_enabled()
+bool AC_AutoTune::roll_enabled()
```

Comment:
If you're going to change these lines then you can probably make them const as well although it's not a blocker

Reply:
done.  thanks!